UM2N/processor/processor.py

```diff
@@ -365,6 +357,28 @@ def to_train_data(self):
             "swirl_params": self.swirl_params,
             "t": self.t,  # time step when solving burgers eq.
             "idx": self.idx,  # index number for picking params for burgers tracer.  # noqa
+            "u": self.feature["uh"] if "uh" in self.feature else None,
```

Comment:
This is equivalent to the following. If you could replicate this for the others, that'd be great.
```suggestion
            "u": self.feature.get("uh"),
```